__tests__/app.test.js

```diff
@@ -69,7 +68,39 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/kestrels")
       .expect(400)
       .then((res) => {
-        expect(res.body.msg).toEqual("invalid input");
+        expect(res.body.msg).toEqual("Bad request");
+      });
+  });
+});
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("Status 200 - responds with updated article object", () => {
+    const articleUpdate = { inc_votes: 1 };
+    return request(app)
+      .patch("/api/articles/1")
+      .send(articleUpdate)
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article.article_id).toEqual(1);
```

Comment:
I think you would need to add a little more to this test to assert the update has actually taken place - if you make a similar assertion to getting the article by id - you could put emphasis on the `votes` property increasing